### PR TITLE
New profile: qt6ct

### PR DESCRIPTION
**Pull request by glitsj16**

### CODE REVIEW

File: etc/profile-m-z/qt6ct.profile [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# Persistent local customizations
+include qt6ct.local
+# Persistent global definitions
+ignore dns
+include globals.local
+
+blacklist /usr/libexec
+
+noblacklist ${HOME}/.config/qt6ct
+noblacklist ${HOME}/.local/share/qt6ct
```
Comment (kmk3) [inline etc/profile-m-z/qt6ct.profile:13]: Not blacklisted.

Comment (kmk3) [reply etc/profile-m-z/qt6ct.profile]: Sorry, I meant add the paths to disable-programs.inc rather than remove.

**Review by kmk3 — COMMENTED** [review]
@glitsj16

Note: This profile is nearly identical to the one in #6249 (other than the
program name and paths using it), so most review comments are likely to be
identical or very similar.

In such cases in future PRs please use the same PR for all similar profiles to
make reviews easier.

**Comment by glitsj16** [comment]
> Sorry, I meant add the paths to disable-programs.inc rather than remove.

Oh, I misunderstood that. Additionally I got confused by there `not` being a counter-part blacklist ${HOME}/.config/qt{5,6}ct anywhere to the below:

https://github.com/netblue30/firejail/blob/df257a814245c8457f1334a01752117aca04d088/etc/inc/whitelist-1793-workaround.inc#L25-L26

So indeed, it would make sense to add those. I'll take care of it.

> In such cases in future PRs please use the same PR for all similar profiles to
make reviews easier.

Will keep it in mind!

File: etc/profile-m-z/qt6ct.profile (hunk 1 above)
Comment (kmk3) [reply etc/profile-m-z/qt6ct.profile]: These lines should be restored.

Comment (kmk3) [reply etc/profile-m-z/qt6ct.profile:13]: > These lines should be restored.

Without removing the `blacklist` ones, as both are needed.

Anyway, I fixed and force-pushed.

File: etc/profile-m-z/qt6ct.profile [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Firejail profile for qt6ct
+# Description: Qt6 Configuration Utility
+# This file is overwritten after every install/update
+# Persistent local customizations
+include qt6ct.local
+# Persistent global definitions
+ignore dns
```
Comment (kmk3) [inline etc/profile-m-z/qt6ct.profile:7]: I just noticed this, why is it here?

There is no `dns` command in any profile.

Does this tool need internet access?

**Review by kmk3 — APPROVED** [review]

**Pull request merged**